Lib/test/test_generators.py

```diff
@@ -170,6 +170,21 @@ def f():
             g.send(0)
         self.assertEqual(next(g), 1)
 
+    def test_handle_frame_object_in_creation(self):
+
+        def cb(*args):
+            try:
+                sys._getframe(1)
+            except ValueError:
+                pass
+
+        gc.set_threshold(1, 0, 0)
```

Comment:
This will modify the `gc` threshold for the whole test suite. Either you reset the original one in a `finally` block or we find another way to trigger the bug (maybe the example in the original issue).

Reply:
Both examples in the original issue use `gc.set_threshold(1, 0, 0)`. I've wrapped it in a try-finally to restore the original value.